controllers/list.controller.js

```diff
@@ -1,10 +1,20 @@
 const { list } = require("../models");
 
 exports.add = (req, res) => {
-  if (!req.body["name"].trim()) {
+  const { name } = req.body;
+  const { price } = req.body;
+  if (name == "" || name == undefined) {
     res.status(422);
     res.send({answer: "Shop name is required!"});
     return 0;
+  } else if (price == null || price < 0 || price == "" || price == undefined) {
```

Comment:
what if i don't send price, name or cost at all. why I need to check conditions if they don't even exist ?